pkg/certificates/renewerbottlerocket.go

```diff
@@ -0,0 +1,358 @@
+apiclient get | apiclient exec admin jq -r '.settings.kubernetes["static-pods"] | keys[]' | xargs -n 1 -I {} apiclient set settings.kubernetes.static-pods.{}.enabled=false
+sleep 10
+apiclient get | apiclient exec admin jq -r '.settings.kubernetes["static-pods"] | keys[]' | xargs -n 1 -I {} apiclient set settings.kubernetes.static-pods.{}.enabled=true
+`
+	return script
+}
+
+func buildBREtcdBackupCertsCmd(backupDir string) string {
+	script := fmt.Sprintf(`cp -r /var/lib/etcd/pki /var/lib/etcd/pki.bak_%[1]s
+rm /var/lib/etcd/pki/*
+cp /var/lib/etcd/pki.bak_%[1]s/ca.* /var/lib/etcd/pki`, backupDir)
+	return script
+}
+
+func buildBREtcdRenewCertsCmd() string {
+	script := `ctr run \
+--mount type=bind,src=/var/lib/etcd/pki,dst=/etc/etcd/pki,options=rbind:rw \
+--net-host \
+--rm \
+${IMAGE_ID} tmp-cert-renew \
+/opt/bin/etcdadm join phase certificates http://eks-a-etcd-dumb-url --init-system kubelet`
+	return script
+}
+
+func buildBREtcdRenewChecksCmd() string {
+	script := `ETCD_CONTAINER_ID=$(ctr -n k8s.io c ls | grep -w "etcd-io" | cut -d " " -f1 | tail -1)
+ctr -n k8s.io t exec --exec-id etcd ${ETCD_CONTAINER_ID} etcdctl \
+     --cacert=/var/lib/etcd/pki/ca.crt \
+     --cert=/var/lib/etcd/pki/server.crt \
+     --key=/var/lib/etcd/pki/server.key \
+     member list`
+	return script
+}
+
+func buildBREtcdCopyCertsToTmpCmd(tempDir string) string {
+	script := fmt.Sprintf(`cp /var/lib/etcd/pki/apiserver-etcd-client.* %[1]s/ || exit 1
+chmod 600 %[1]s/apiserver-etcd-client.crt || exit 1
+chmod 600 %[1]s/apiserver-etcd-client.key || exit 1`, tempDir)
+	return script
+}
+
+func buildBREtcdCleanupTmpCmd(tempDir string) string {
+	script := fmt.Sprintf(`rm -f %s/apiserver-etcd-client.*`, tempDir)
+	return script
+}
+
+func buildBRCreateTmpDirCmd(dirName string) string {
+	script := fmt.Sprintf(`TARGET_DIR="/run/host-containerd/io.containerd.runtime.v2.task/default/admin/rootfs/tmp/%[1]s"
+mkdir -p "${TARGET_DIR}" || exit 1
+chmod 755 "${TARGET_DIR}" || exit 1`, dirName)
+	return script
+}
+
+func buildBRWriteCertToTmpCmd(certBase64 string) string {
+	script := fmt.Sprintf(`cat <<'CRT_END' | base64 -d > "${TARGET_DIR}/apiserver-etcd-client.crt"
+%s
+CRT_END
+[ $? -eq 0 ] || exit 1`, certBase64)
+	return script
+}
+
+func buildBRWriteKeyToTmpCmd(keyBase64 string) string {
+	script := fmt.Sprintf(`cat <<'KEY_END' | base64 -d > "${TARGET_DIR}/apiserver-etcd-client.key"
+%s
+KEY_END
+[ $? -eq 0 ] || exit 1`, keyBase64)
+	return script
+}
+
+func buildBRSetTmpCertPermissionsCmd() string {
+	script := `chmod 600 "${TARGET_DIR}/apiserver-etcd-client.crt" || exit 1
+chmod 600 "${TARGET_DIR}/apiserver-etcd-client.key" || exit 1`
+	return script
+}
+
+func buildBRListTmpFilesCmd(tempDir string) string {
+	script := fmt.Sprintf(`sudo sheltie << 'EOF'
+ls -l %s/apiserver-etcd-client.*
+exit
+EOF`, tempDir)
+	return script
+}
+
+func buildBRReadTmpCertCmd(tempDir string) string {
+	script := fmt.Sprintf(`sudo sheltie << 'EOF'
+cat %s/apiserver-etcd-client.crt
```

Comment:
Can we combine getting data from key and crt file by passing entire file name with folder?